### ISSUE TITLE
Can it run without xys and point3d idx

### ISSUE THREAD

**Issue by Grizemc** [prstrive/SCGaussian#8]
def read_extrinsics_text(path):
    """
    Taken from https://github.com/colmap/colmap/blob/dev/scripts/python/read_write_model.py
    """
    images = {}
    with open(path, "r") as fid:
        while True:
            line = fid.readline()
            if not line:
                break
            line = line.strip()
            if len(line) > 0 and line[0] != "#":
                elems = line.split()
                image_id = int(elems[0])
                qvec = np.array(tuple(map(float, elems[1:5])))
                tvec = np.array(tuple(map(float, elems[5:8])))
                camera_id = int(elems[8])
                image_name = elems[9]
                images[image_id] = Image(
                        id=image_id, qvec=qvec, tvec=tvec,
                        camera_id=camera_id, name=image_name)
                elems = fid.readline().split()
                xys = np.column_stack([tuple(map(float, elems[0::3])),
                                       tuple(map(float, elems[1::3]))])
                point3D_ids = np.array(tuple(map(int, elems[2::3])))
                images[image_id] = Image(
                    id=image_id, qvec=qvec, tvec=tvec,
                    camera_id=camera_id, name=image_name,
                    xys=xys, point3D_ids=point3D_ids)

I only have four images, but these four images cannot be run with colmap to obtain results. However, I have the extrinsic and intrinsic parameters of the camera and point cloud files. But there is no 2D/3D matching point file, which means there is no data for xys and points3didx. Can I still use your algorithm？

**Comment by prstrive** [comment]
Yes, we do not use "xys" and "point3D_ids" properties in the project.
